api/src/db/models/user_models.py

```diff
@@ -162,3 +163,81 @@ class UserOpportunityNotificationLog(ApiSchemaTable, TimestampMixin):
     opportunity: Mapped[Opportunity] = relationship(
         "Opportunity", back_populates="all_opportunity_notification_logs"
     )
+
+
+class StagingUserAccountMapper(Base):
+    __tablename__ = "TUSER_ACCOUNT_MAPPER"
+    __table_args__ = {"schema": Schemas.STAGING}
+
+    USER_ACCOUNT_ID: Mapped[int] = mapped_column(primary_key=True)
```

Comment:
FYI - for these, we can make them lowercase, Postgres is case-insensitive so we can at least have some err on the side of readability without any issue.